src/lib.rs

```diff
@@ -11,6 +11,6 @@ pub use args::Args;
 pub use artifact::{Artifact, ArtifactType};
 pub use config::{EnvError, EnvOption, LocalizedConfig};
 pub use error::Error;
-pub use manifest::CrateType;
+pub use manifest::{CrateType, Manifest};
```

Comment:
Getting complains that one field is not read inside `Manifest` -> `Workspace`. We don't expose any public API with `Manifest` yet but might as well export it.

Or should we mark it all `pub(crate)`?